[CAAS| VM] Sepolicy Addition for file sharing

taranpreetkaur0013 opened the pull request:
Tracked-On: OAM-88852

Signed-off-by: tkaur <taranpreet.kaur@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/1475

taranpreetkaur0013 committed:
Sepolicy Addition for file sharing

Sepolicy addition for file edit in VM guest OSes.
For multi-VM environment

Tracked-On: OAM-89748
Signed-off-by: tkaur <taranpreet.kaur@intel.com>
